ir/func.go

```diff
@@ -132,28 +136,23 @@ func (f *Func) LLString() string {
 		}
 		buf.WriteString(headerString(f))
 		return buf.String()
+	} else {
```

Comment:
Since the previous `if` statement always ends with a `return` statement, we can safely remove this `else` statement and thus reduce the indentation level of the succeeding code. (This is related to but not identical with the suggestion given in https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow)

Reply:
I understand the early return style, but IMO early return shouldn't use here.

Here have two same level concepts, explicitly point out would be better.

Reply:
> Here have two same level concepts, explicitly point out would be better.

Ok, you have me convinced :)